packages/web-components/public/switches.ts

```diff
@@ -0,0 +1,25 @@
+import { fillColor, StandardLuminance, SwatchRGB } from '../src/index';
+
+export function toggleBgMode() {
+  const bgChecked = document.getElementById('luminance-switch')!.classList.contains('checked');
+  if (bgChecked) {
+    fillColor.setValueFor(
```

Comment:
You'll want to change this over to `baseLayerLuminance` with my recent PR once it's published. `fillColor` will default to the neutral layer 1 recipe.